debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -462,4 +468,13 @@ private int getSqlServerVersion() {
             throw new RuntimeException("Couldn't obtain database server version", e);
         }
     }
+
+    @Override
+    protected void setDefaultValue(ColumnEditor columnEditor, String defaultValue) {
```

Comment:
Couldn't this happen in the super-class, by passing the default value converter in the constructor?

Reply:
I tried to implement it and there is a circular dependency between JdbcConnection and DefaultValueConverter. The net result is that I can't call `super` and use `this` in it.

Reply:
Ok, got it. So I have to admit I'm still not 100% with this:

* Leaking an in-flight `ColumnEditor`; can we avoid that? What's really needed from it?
* Can that method simply _return_ the value, leaving the mutating call to the editor to the base connection class?